### PR TITLE
Renders code preview with correct import relationship

### PR DESCRIPTION
**Pull request by ibeeliot**
**Problem:** 'Native' child type was written without regression towards child types 'Comp' and 'HTML' so properties for 'COMP' were not being recognized in logic chain for rendering 'COMP' components

**Solution:** Corrected conditionals in component render and code preview to make explicit child types that were 'COMP' and to render those components correctly while also observing the values of other child types.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -144,6 +143,7 @@
     "react-dom": "^16.4.1",
     "react-draggable": "^3.0.5",
     "react-konva": "^16.12.0-0",
+    "react-native": "^0.62.1",
```
Comment (tonyito) [inline package.json:146]: Why is react native installed? I think we should remove this.

File: src/reducers/leftReducers.ts [hunk 2]
```diff
@@ -302,6 +302,7 @@ export const deleteChild = (
     however when deleting  component we wnt to delete ALL the places where it's used, so we call this function
     Also when calling from DELETE components , we do not touch focusComponent.
    ************************************************************************************ */
+  console.log('TESTTTTTTTTTTTTTTT');
```
Comment (tonyito) [inline src/reducers/leftReducers.ts:305]: Remove this please

Comment (ibeeliot) [reply src/reducers/leftReducers.ts]: removed. Thanks!

File: package.json (hunk 1 above)
Comment (ibeeliot) [reply package.json]: removed. Thanks!

**Pull request merged**